person.rb

```diff
@@ -29,14 +29,15 @@ def correct_name
 class Person < Nameable
   attr_reader :id
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name, parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] A persone has many rentales kindly consider adding this relation to the person class
e.g
```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
 end
```